hack/make-rules/deploy.sh

```diff
@@ -44,4 +45,4 @@ fi
 terraform -v
 terraform init
 # NOTE: this must use :? expansion to ensure we will not run with unset variables
-terraform apply -auto-approve -var digest="${IMAGE_DIGEST:?}"
+(set -x; terraform apply -auto-approve -var digest="${IMAGE_DIGEST:?}")
```

Comment:
this ensures we log exactly what we ran including the supplied `-var digest=$digest`